### PR TITLE
Cobbler support statement

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://bugzilla.suse.com/show_bug.cgi?id=1206963
https://github.com/SUSE/spacewalk/issues/20083

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by SchoolGuy — APPROVED** [review]
LGTM

File: modules/client-configuration/pages/autoinst-intro.adoc [hunk 1]
```diff
@@ -41,8 +41,11 @@ Autoinstallation of {ubuntu} and {debian} clients is not supported.
 These operating systems must be installed manually.
 ====
 
+The autoinstallation features of {productname} are based on a software named Cobbler.
+For more information about cobbler, see https://cobbler.readthedocs.io.
```
Comment (nodeg) [inline modules/client-configuration/pages/autoinst-intro.adoc:45]:
```suggestion
For more information about Cobbler, see https://cobbler.readthedocs.io.
```

**Review by nodeg — CHANGES_REQUESTED** [review]
Cobbler is wirtten with a capital C and also to be consistent with the line above.

**Comment by keichwa** [comment]
> Cobbler is wirtten with a capital C and also to be consistent with the line above.

Thanks, good point.  I should probably check the complete file or even all the books in a separate PR.

**Review by nodeg — APPROVED** [review]

**Pull request merged**